libs/labelbox/src/labelbox/schema/project_overview.py

```diff
@@ -0,0 +1,18 @@
+from typing import Dict, Optional
+from labelbox import pydantic_compat
+
+class ProjectOverview(pydantic_compat.BaseModel):
```

Comment:
Do you just want to import the BaseModel, small style knit, also use code pilot to generate the param definitions?

On that note, it's probably worth updating the docs/.rst to include this new project so it shows up on readthedocs

Reply:
Thanks for the comment.

All the attributes are now documented.
`project-overview.rst` was added.

I am not sure of what you mean by "_Do you just want to import the BaseModel, small style knit,_". Are you suggesting that a dataclass would be more appropriate in this case?

Reply:
`from labelbox.pydantic_compat import BaseModel`

Reply:
OK but `from labelbox import pydantic_compat` is used everywhere else. I can change it's going to look off.

![image](https://github.com/Labelbox/labelbox-python/assets/87332996/f7a41fc7-08c0-4f04-b2a7-28225e2a6fd7)

Reply:
Import changed.